src/sweeper/report.py

```diff
@@ -5,29 +5,41 @@
 A module that contains the templates for the reports and functions to format data into the report
 '''
 
-template = '''# Sweeper Empty Report
-
-**{0}** empty geometries found
-
-## ObjectId's of empty geometries
-
-`{1}`
-
-## Select features in arc
-
-`objectid in ({1})`
-'''
-
-success_template = '''# Sweeper Empty Report
-
-🚀🚀🚀 No empty geometries found 🚀🚀🚀
-
-
-if len(report.oids) > 0:
-  return string.format(template, len(report.oids), ','.join(report.oids))
-
-return success
-
-
-'sweeper-report-{}.{}.txt'.format(date.short, report_type)
-'''
+def save_report(data_dict, title, fc, path):
+    import os
+    from datetime import datetime
+    
+    now = datetime.now().strftime('%Y%m%d_%H%M%S_%f')[:-3]
+    name = f'{title}_report_{now}.txt'
+    filename = os.path.join(path, name)
+    if len(data_dict) > 0:
+        lines = []
+        with open(filename, 'w') as file:
+            file.write(f"{title.title()} Report for '{fc}' \n \n")
+            file.write(f'{len(data_dict)} empty geometries found in feature class \n \n')
+            file.write('Issues found: \n')
+            for k,v in data_dict.items():
+                lines.append(f'ObjectID {k}: {v} \n')
+            file.writelines(lines)
+            file.write('\n')
+            file.write('Select statement to view issues in ArcGIS: \n')
+            statement = 'OBJECTID IN ({})'.format(','.join(str(key) for key in data_dict))
+            file.write(statement)
+    else:
+        with open(filename, 'w') as file:
+            file.write(f"{title.title()} Report for '{fc}' \n \n")
+            file.write('No empty geometries found!')
+            
+def print_report(data_dict, title, fc):
+    if len(data_dict) > 0:
```

Comment:
same as above with removing the else. Also you can pull the report title out and have it above the first if since it is common in both reports. That keeps it DRY which means do not repeat yourself.